app/lib/frontend/templates/views/publisher/admin_page.mustache

```diff
@@ -13,7 +13,7 @@
 </div>
 
 <div id="-admin-authorized">
-  <h4>Description</h4>
+  <h4>Description (markdown)</h4>
```

Comment:
Do you want to link to a description of markdown?

Reply:
Not here... maybe when we do this materialized...